templates/_components/_image.twig

```diff
@@ -5,7 +5,7 @@
   
   Generate, transform, manipulate, optimize and render images. 
 
-	What it can do:
+	What it does:
 	---------------
```

Comment:
Remove the colon at the end (it is already a heading) and reduce the dashes `---` to end where the heading text ends.

Reply:
Fixed.